### PR TITLE
final copy

### PR DESCRIPTION
**Pull request by greggb**
Approved copy from the meeting yesterday. Also includes a few style tweaks that were called out.

### CODE REVIEW

File: src/pages/community/index.js [hunk 1]
```diff
@@ -169,12 +169,14 @@ export default ({ ...data }) => {
                             around the globe.
                         </P>
                         <ProjectActions>
-                            <Button to="#" secondary>
+                            <Button to="/article" primary>
```
Comment (jestapinski) [inline src/pages/community/index.js]:
```suggestion
                            <Button to="/learn" primary>
```

Should this go to the learn page?

File: src/pages/community/index.js [hunk 2]
```diff
@@ -169,12 +169,14 @@ export default ({ ...data }) => {
...
                                 Learn how they did it
                             </Button>
                             <ProjectSignUpForm
                                 triggerComponent={
-                                    <Link tertiary>Share your project</Link>
+                                    <Button secondary>
```
Comment (jestapinski) [inline src/pages/community/index.js]: We might want some spacing between these buttons?
Screen Shot 2020-02-27 at 3 10 48 PM

File: src/pages/index.js [hunk 3]
```diff
@@ -92,36 +92,35 @@ export default ({ ...data }) => {
...
-                    {`ideas.find({"attributes" : ["fast", "innovative",
-                    "original"]})`}
+                    {`ideas.find({"attributes":`}
+                    <br />
+                    {`["fast", "innovative", "original"]})`}
                 </Heading>
                 <Sub>What will you create today?</Sub>
                 <CardGallery>
-                    <StyledTopCard image={unityImage}>
+                    <StyledTopCard image={unityImage} to="/article">
```
Comment (jestapinski) [inline src/pages/index.js]: Similarly, should these go to the learn page? Looks like `/article` has moved so it 404s for me.

File: src/pages/community/index.js (hunk 1 above)
Comment (greggb) [reply src/pages/community/index.js]: I think it's going to go to a specific article by Radar

File: src/pages/index.js (hunk 3 above)
Comment (greggb) [reply src/pages/index.js]: They'll go to specific articles

**Comment by greggb** [comment]
Will address the button issue in a followup - the filterbar on the events page is breaking the build

**Pull request merged**